README.md

```diff
@@ -7,7 +7,7 @@ The unified API for building rich integrations with multiple Accounting & Financ
 This SDK is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
 
 - API version: 1.0
-- Package version: 1.0.0
+- Package version: 1.0.1
```

Comment:
Do you think this should also go up to major version 2.x to be consistent with the other categories?